src/js/base/model.js

```
@@ -43,9 +44,16 @@ export default Backbone.Model.extend(extend({
       .then(response => {
         this._isFetching = false;
 
+        // NOTE: If the response fails, a retry will include notiications results.
+        if (!response) {
+          delete this._notifications;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)